ui/app/AppLayouts/Wallet/adaptors/CollectiblesSelectionAdaptor.qml

```diff
@@ -32,7 +32,7 @@ QObject {
     /**
       Expected model structure:
 
-        tokenId             [string] - unique identifier of a collectible
+        symbol              [string] - unique identifier of a collectible
```

Comment:
Could we keep the `tokenId` as a unique identifier and add the additional `symbol` property? I'm not sure the `symbol` can be unique in all cases.

Reply:
It would be great to gradually align to the point from https://github.com/status-im/status-desktop/blob/73e86a37bc290116ef7a218bf6a6084a7d6e66e7/architecture-guide.md:

> use consistently key as a unique identifier of the model, even if there are other roles with unique values.
> 
> On the UI side, key should be used only for identification, with no other assumptions and usage. Thanks to that, content of that role can be freely changed on the backend with no implications to the UI side. Even if address is unique, a separate key role should be used (providing the same content as address role).
> 
> Other good name for that role - id - doesn't fit well in qml environment as it's reserved keyword. Defining such role with ListElement in tests or Storybook would be not possible.

Reply:
We can have both `key` and `symbol`. Both roles may even have the same value under the hood. But UI should never assume anything more about `key` than uniqueness and string type. Conversely, there should be no assumption that `symbol` is unique.